db/seeds/evaluatieonderzoek/questionnaires/evaluatieonderzoek.rb

```diff
@@ -1,85 +1,945 @@
+                       labels: ['Veel minder kwaliteit', 'Veel meer kwaliteit'],
+                       tooltip: maatregelitje,
+                     }, {
+                       id: :v13_1,
+                       type: :textarea,
+                       title: 'Eventuele toelichting (bij bovenstaand antwoord)',
+                       placeholder: 'Vul hier een eventuele toelichting in'
+                     }, {
+                       id: :v14,
+                       type: :range,
+                       title: 'Hoe ervaart u de kwaliteit van de monitoring/begeleiding van jongeren vergeleken met de periode voor de invoering van de beleidsmaatregelen in 2016?',
+                       labels: ['Veel minder kwaliteit', 'Veel meer kwaliteit'],
+                       tooltip: maatregelitje
+                     }, {
+                       id: :v14_1,
+                       type: :textarea,
+                       title: 'Eventuele toelichting (bij bovenstaand antwoord)',
+                       placeholder: 'Vul hier een eventuele toelichting in'
+                     }, {
+                       id: :v15,
+                       hidden: true,
+                       type: :range,
+                       title: 'Is er verandering gekomen in de hoeveelheid jongeren die u monitort/begeleidt vanaf het <strong>praktijkgericht onderwijs (PrO)</strong>, sinds de invoering van de beleidsmaatregelen in 2016?',
+                       labels: ['Veel minder jongeren van PrO', 'Veel meer jongeren van PrO'],
+                       tooltip: maatregelitje
+                     }, {
+                       id: :v16,
+                       hidden: true,
+                       type: :range,
+                       title: 'Is er verandering gekomen in de hoeveelheid jongeren die u monitort/begeleidt vanaf het <strong>voortgezet speciaal onderwijs (VSO)</strong>, sinds de invoering van de beleidsmaatregelen in 2016?',
+                       labels: ['Veel minder jongeren van VSO', 'Veel meer jongeren van VSO'],
+                       tooltip: maatregelitje,
+                     }, {
+                       section_start: '',
+                       type: :raw,
+                       content: '<div class="divider"></div><p class="flow-text"><strong>Wilt u onderstaande deel-maatregelen op enkele aspecten evalueren?</strong></p><p class="flow-text"><strong>Maatregel 1</strong>: de regionale samenwerking wordt geïntensiveerd en uitgebreid om een sluitend vangnet te creëren voor jongeren in een kwetsbare positie, afkomstig uit het praktijkonderwijs en voortgezet speciaal onderwijs. <a onclick="M.toast({html: &#39;Voor deze jongeren moet in de regio een sluitend vangnet zijn tussen gemeenten (RMC en Dienst Werk &amp; Inkomen), onderwijs (PrO, VSO, VO, MBO), (jeugd)zorg en werkgevers/arbeidsmarktdomein (zoals UWV, jongerenloket, servicepunt arbeid).&#39;}, 6000)"><i class="tooltip flow-text material-icons info-outline">info</i></a></p>',
```

Comment:
blegh. We moeten ook aan de raw type een tooltip kunnen toevoegen eigenlijk. Al is dat misschien ook wel weer lastig met plaatjes e.d.. Wat denk jij?

Reply:
Ja lelijk he. Eigenlijk moeten we raw question types niet te veel gebruiken. Ook omdat ze geen ID hebben en dus niet gehide kunnen worden. Later in de vragenlijst avoid ik raw question types helemaal door een heleboel spul in de section_start te zetten en die te `html_safe`en. Ik zou zeggen gewoon zo min mogelijk raw question types gebruiken omdat ze geen id hebben en we ze niet kunnen hiden en ze dus ook niet geschikt zijn voor expandables en vragenlijsten waar delen standaard van gehide zijn. Misschien kunnen we op termijn af van de raw question types door alles in section_starts te zetten, maar dat moeten we nog even zien. Voor nu zo laten?